akka-stream/src/main/scala/akka/stream/Materializer.scala

```diff
@@ -101,4 +101,4 @@ private[akka] object NoMaterializer extends Materializer {
 case class MaterializationContext(
   materializer:        Materializer,
   effectiveAttributes: Attributes,
-  stageName:           String)
+  islandName:          String)
```

Comment:
this is not binary compatible change, this is grey zone public so I guess it's fine